### ISSUE TITLE
Restore f behavior and merge interactive filter prefixes to F

### ISSUE THREAD

**Issue by sei40kr** [wsdjeg/SpaceVim#929]
`f` is originally a key to find any character in current line, but currently it's overridden by Unite prefix.
Since this is frequently used feature in Vim, I would like to restore original `f` behavior (or install `rhysd/clever-f` by default).

In addition to this, Denite includes most of the features Unite has, and it avails only on NeoVim.
So if the source and Denite both avail, **the user won't have any reason to use Unite**.
In other words, there's no need to define 2 prefix keys for Unite and Denite (overriding Unite mappings with Denite mappings should be enough).

**Comment by wsdjeg** [comment]
to find content I recommend to use `SPC j j` it is better than origin `f`, `f` `s` `F` is the easiest key which user can press, I think use this as `origin f` is not a goog idea. NOTE, `not all vim's default key binding is the best choice`.

> I never use `origin f` in last 5 years.

Denite need python3 support, but spacevim can works well without py3.

**Comment by sei40kr** [comment]
Sorry I didn't noticed that original `f` has been remapped to `Leader f`.
(I think `f` behavior of default Vim is not useful too, but `clever-f` is.)
Closing this ticket.

> Denite need python3 support

I know it and that Denite have fewer sources too.
But I think no user will be going to use Unite source when same source avails on Denite,
so if the prefixes are merged, users will be able to use them without thinking about which filter to use.

**Comment by wsdjeg** [comment]
yeah, I will try to merge the prefixes of denite and unite
